### DISCUSSION TITLE
Plan/feasibility for support Envoy proxy with bucket4j

### DISCUSSION THREAD

**Discussion by vinaypareek** [vladimir-bukhtoyarov/bucket4j#395] [General]
Is there any roadmap to support envoy proxy https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_protocols/redis as well with bucket4j.

And what do you think, does functionality of Bucket4j will work in any multi data centre deployment? I was looking integration with redis, also Is there any other integration of bucket4j that already supporting this (multi data centre deployment).

**Comment by vladimir-bukhtoyarov** [comment]
Hello @vinaypareek

> Is there any roadmap to support envoy proxy

I suppose that it should work with enoy-proxy, but I have no time to test.

> And what do you think, does functionality of Bucket4j will work in any multi data centre deployment?

Bucket4j should support all scenarios that is supported by [Lettuce](https://github.com/lettuce-io).
